packages/components/src/internal/components/domainproperties/list/ListDesignerPanels.tsx

```diff
@@ -86,6 +93,33 @@ class ListDesignerPanelsImpl extends React.PureComponent<Props & InjectedBaseDom
         this.setState({ file, shouldImportData });
     };
 
+    onImportErrorStayAndFix = () => {
+        const { model } = this.state;
+
+        this.setState({importErrorModalOpen: false});
+        const dropOptions = {
+            schemaName: 'lists',
+            queryName: model.name,
+            failure: (error) => {
+                const updatedModel = model.set('exception', error) as ListModel;
+                this.setState({ model: updatedModel });
+            },
+            success: () => {
+                this.setState((state) => ({ model: model.merge({
+                        domain: state.preSaveDomain,
+                        'exception': undefined
+                }) as ListModel }), () => {
+                    this.setState({preSaveDomain: undefined});
```

Comment:
same comment here, can we just set this `preSaveDomain` state property as part of line 108?

Reply:
Done!